packages/client/components/ReflectionCard/ReflectionCard.tsx

```diff
@@ -179,41 +224,89 @@ const ReflectionCard = (props: Props) => {
+  }
+
+  const showSpotlight = true // TODO: dummy feature flag. Change to false before merging PR
+  const showSearch =
+    phaseType === 'group' &&
+    !inSpotlight &&
+    !isComplete &&
+    showSpotlight &&
+    (isHovering || !isDesktop)
+  if (!meeting) return null
   return (
-    <ReflectionCardRoot data-cy={`${dataCy}-root`}>
-      <ColorBadge phaseType={phaseType as NewMeetingPhaseTypeEnum} reflection={reflection} />
-      <ReflectionEditorWrapper
-        dataCy={`editor-wrapper`}
-        isClipped={isClipped}
-        ariaLabel='Edit this reflection'
-        editorRef={editorRef}
-        editorState={editorState}
-        onBlur={handleEditorBlur}
-        onFocus={handleEditorFocus}
-        handleReturn={handleReturn}
-        handleKeyDownFallback={handleKeyDownFallback}
-        placeholder={isViewerCreator ? 'My reflection… (press enter to add)' : '*New Reflection*'}
-        readOnly={readOnly}
-        setEditorState={setEditorState}
-        userSelect={userSelect}
-      />
-      {error && <StyledError onClick={clearError}>{error.message}</StyledError>}
-      {!readOnly && (
-        <ReflectionCardDeleteButton
-          dataCy={`reflection-delete`}
-          meetingId={meetingId}
-          reflectionId={reflectionId}
+    <>
+      <ReflectionCardRoot
+        data-cy={`${dataCy}-root`}
+        onMouseEnter={() => setIsHovering(true)}
+        onMouseLeave={() => setIsHovering(false)}
+      >
+        <ColorBadge phaseType={phaseType as NewMeetingPhaseTypeEnum} reflection={reflection} />
+        <ReflectionEditorWrapper
+          dataCy={`editor-wrapper`}
+          isClipped={isClipped}
+          ariaLabel='Edit this reflection'
+          editorRef={editorRef}
+          editorState={editorState}
+          onBlur={handleEditorBlur}
+          onFocus={handleEditorFocus}
+          handleReturn={handleReturn}
+          handleKeyDownFallback={handleKeyDownFallback}
+          placeholder={isViewerCreator ? 'My reflection… (press enter to add)' : '*New Reflection*'}
+          readOnly={readOnly}
+          setEditorState={setEditorState}
+          userSelect={userSelect}
         />
+        {error && <StyledError onClick={clearError}>{error.message}</StyledError>}
+        {!readOnly && (
+          <ReflectionCardDeleteButton
+            dataCy={`reflection-delete`}
+            meetingId={meetingId}
+            reflectionId={reflectionId}
+          />
+        )}
+        {showReactji && <StyledReacjis reactjis={reactjis} onToggle={onToggleReactji} />}
+        <ColorBadge phaseType={phaseType as NewMeetingPhaseTypeEnum} reflection={reflection} />
+        <SearchButton
+          onClick={openSpotlight}
+          onMouseEnter={openTooltip}
+          onMouseLeave={closeTooltip}
+          showSearch={showSearch}
+        >
+          <SearchIcon ref={tooltipRef} icon='search' />
+        </SearchButton>
+        {tooltipPortal('Find similar')}
+      </ReflectionCardRoot>
+      {modalPortal(
```

Comment:
let's move this logic to the RetroGroupPhase or GroupingKanban. then we can pass down something like:
```ts
const openSpotlight = (reflection: Reflection) => {
  commitLocalUpdate(atmosphere, (store) => store.get(meetingId).setLinkedRecord(reflection, 'spotlightReflection')
})
```
then we only open the modal if it was falsy & now it's a reflection object.
that'll express that there can only be 1 spotlight open & a spotlight takes in a reflection.